paddle/fluid/imperative/tests/test_prepare_op.cc

```diff
@@ -215,4 +225,4 @@ TEST(test_prepare_op, test_prepare_data_same_place) {
 }  // namespace paddle
 
 USE_OP(split);
```

Comment:
Why `test_prepare_op` change usage from `assign_op` to `relu_op`?

Reply:
Because the previous implementation of the unittest is wrong. Previously, it use `OperatorWithKernel` to create `assign` op, so that the virtual method (for example `GetKernelTypeForVar`) called is from `OperatorWithKernel` but not from `SplitOp`.